### ISSUE TITLE
Feature request: Require fat arrow for anonymous functions

### ISSUE THREAD

**Issue by scriby** [palantir/tslint#119]
Disallowed:

``` javascript
fs.readFile('path', 'utf8', function(err, contents) {

});
```

Allowed:

``` javascript
fs.readFile('path', 'utf8', (er, contents) => {

});
```
